pkg/cri/sbserver/podsandbox/controller.go

```diff
@@ -123,7 +139,7 @@ func (c *Controller) waitSandboxExit(ctx context.Context, id string, exitCh <-ch
 
 			sb, err := c.sandboxStore.Get(id)
 			if err == nil {
-				if err := handleSandboxExit(dctx, sb); err != nil {
+				if err := handleSandboxExit(dctx, sb, &eventtypes.TaskExit{ExitStatus: exitStatus, ExitedAt: protobuf.ToTimestamp(exitedAt)}); err != nil {
```

Comment:
nod to @fuweid 's suggestion to add new api params to the end vs insert new/additional fields in between..  I see you already did that!

Additionally it feels(reads) wrong to create a partial task exit event for a sandbox then only use the new fields.. and discard the task exit event.. maybe just pass in the new fields as params? ExitedAt time.Time and ExitStatus uint32